src/runtime/hexagon_remote/halide_host.cpp

```diff
@@ -237,13 +237,26 @@ void halide_hexagon_host_free(void *ptr) {
 
     free(rec);
 }
-
 // This is a shim for calling v2 from v1.
 handle_t halide_hexagon_remote_get_symbol(handle_t module_ptr,
                                           const char* name, int nameLen) {
     handle_t sym = 0;
-    int result = halide_hexagon_remote_get_symbol_v2(module_ptr, name, nameLen, &sym);
+    int result = halide_hexagon_remote_get_symbol_v4(module_ptr, name, nameLen, false, &sym);
     return result == 0 ? sym : 0;
 }
 
+handle_t halide_hexagon_remote_get_symbol_v3(handle_t module_ptr,
```

Comment:
I see the motivation here to make it so get_symbol returns the symbol (instead of using an output parameter) but it would be better to just make one new function halide_hexagon_remote_get_symbol_v3 that has an output parameter (so it can be the definition in the IDL), and just call that directly. I suppose I set a bad example because I used the old API and just made a shim in the runtime, but that was just a bit convenient in that context. Now that we need an entirely new API, we should just make the function that produces an output parameter be the API that is exposed both in the runtime and used in hexagon_host.cpp.